config/schedule.rb

```diff
@@ -42,3 +42,15 @@
 every 1.week, roles: [:cron_staging, :cron_production] do
   rake "marc_liberation:delete:events"
 end
+
+every 2.weeks, at: '7:00am', roles: [:cron_production] do
+  rake "scsb:request_records[production,mk8066@princeton.edu,CUL]", output: "/tmp/cron_log.log"
```

Comment:
Unfortunately the output from the API is pretty minimal, so having the email notifications is really the only way to get a record of what the job does. I'm thinking we should get a service account like Carolyn suggested to receive them so we don't have one our addresses hardcoded in the config.